tss-esapi/src/tcti.rs

```diff
@@ -36,6 +37,30 @@ pub enum Tcti {
     Tabrmd(TabrmdConfig),
 }
 
+impl Tcti {
+    /// Gets a TCTI from the following environment variables, in order:
+    /// - TPM2TOOLS_TCTI
```

Comment:
What is the reasoning behind this order?

I wonder if we should prefix our own environment variables with something to perhaps avoid name collision?

Reply:
The order was random, based on what I thought would be okayish. The primary reason for this function is the doctests.